app/models/filters/filter_base.rb

```diff
@@ -125,42 +128,42 @@ def update(filters) # rubocop:disable Metrics/CyclomaticComplexity, Metrics/Perc
         self.heads_of_household = filter_hoh
         self.hoh_only = filter_hoh
       end
-      self.default_project_type_codes = Array.wrap(filters.dig(:default_project_type_codes))&.reject(&:blank?) if filters.key?(:default_project_type_codes)
+      self.default_project_type_codes = cleanup_input_array(filters.dig(:default_project_type_codes), input_type: :to_s, default: default_project_type_codes) if filters.key?(:default_project_type_codes)
```

Comment:
I tried to DRY this up a bit, I think it's easier to read, but would love some confirmation.